cynthion/python/src/gateware/platform/core.py

```diff
@@ -76,7 +76,7 @@ def toolchain_program(self, products, name):
             programmer.configure(bitstream)
 
         # Let the LUNA gateware take over in devices with shared USB port
-        debugger.honor_fpga_adv()
+        # debugger.honor_fpga_adv()
```

Comment:
Rather than commenting it out can you do a:

```python
try:
    debugger.honor_fpga_adv()
except AttributeError:
    pass
```

Reply:
Good call!

Reply:
Good call!